airflow/providers/asana/hooks/asana.py

```diff
@@ -23,7 +23,7 @@
 from asana.error import NotFoundError
 
 try:
-    from functools import cached_property
+    from functools import cached_property  # type: ignore
```

Comment:
`if sys.version_info >= (3, 8):
    from functools import cached_property
else:
    from cached_property import cached_property`
The sys.version_info didnt fix the mypy error for me(Im trying in breeze), should I be adding the # type: ignore to the sys.version_info code?

Reply:
Not sure if @potiuk also did the same here - https://github.com/apache/airflow/pull/20223/files , because the sys.version.info change doesnt fix the mypy errors.

Reply:
Hmm. Adding the version check worked for me in Breeze.

Reply:
Actually it worked now, maybe I still had the try/catch before